sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2189,6 +2189,24 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
     }
   }
 
+  test("SPARK-17849: grouping set throws NPE") {
```

Comment:
maybe we can move this into SQLQueryTestSuite, by creating a new grouping_set.q file??